src/types/FilterTodosBy/index.ts

```diff
@@ -0,0 +1 @@
+export * from './FilterTodosBy';
```

Comment:
It's recommended to use a clear way to import and a clear way to export to maintain consistency. Instead of using `export * from`, consider explicitly importing and exporting the required elements. 🧐